local-tests/setup/tinny-environment.ts

```diff
@@ -0,0 +1,366 @@
+   *
+   * Logs the process and exits if the client is not ready after attempting to connect.
+   */
+
+  async setupLitNodeClient() {
+    console.log('[𐬺🧪 Tinny Environment𐬺] Setting up LitNodeClient');
+
+    if (this.network === LIT_TESTNET.LOCALCHAIN) {
+      this.litNodeClient = new LitNodeClient({
+        litNetwork: 'custom',
+        bootstrapUrls: this.processEnvs.BOOTSTRAP_URLS,
+        rpcUrl: this.processEnvs.LIT_RPC_URL,
+        debug: this.processEnvs.DEBUG,
+        checkNodeAttestation: false, // disable node attestation check for local testing
+        contractContext: networkContext as LitContractContext,
+      });
+    } else if (this.network === LIT_TESTNET.MANZANO) {
+      this.litNodeClient = new LitNodeClient({
+        litNetwork: this.network, // 'habanero' or 'manzano'
+        checkNodeAttestation: true,
+        debug: this.processEnvs.DEBUG,
+      });
+    } else {
+      this.litNodeClient = new LitNodeClient({
+        litNetwork: this.network,
+        checkNodeAttestation: false,
+        debug: this.processEnvs.DEBUG,
+      });
+    }
+
+    await this.litNodeClient.connect();
+
+    if (!this.litNodeClient.ready) {
+      console.error('❌ litNodeClient not ready');
+      process.exit();
+    }
+  }
+
+  /**
+   * Retrieves the environment configuration.
+   * @returns The TinnyEnvConfig object containing the environment configuration.
+   */
+  getEnvConfig(): TinnyEnvConfig {
+    return {
+      rpc: this.rpc,
+      litNodeClient: this.litNodeClient,
+      network: this.network,
+      processEnvs: this.processEnvs,
+    };
+  }
+
+  /**
+   * Creates a new person with the given name.
+   * @param name - The name of the person.
+   * @returns The newly created person.
+   * @throws Error if the name is not provided.
+   */
+  async createNewPerson(name: string) {
+    console.log('[𐬺🧪 Tinny Environment𐬺] Creating new person:', name);
+    if (!name) {
+      throw new Error('Name is required');
+    }
+    const key = await this.getAvailablePrivateKey();
+    const privateKey = key.privateKey;
+    const envConfig = this.getEnvConfig();
+
+    console.log(
+      '[𐬺🧪 Tinny Environment𐬺] Creating new person with private key:',
+      JSON.stringify(key)
+    );
+    console.log(
+      '[𐬺🧪 Tinny Environment𐬺] Creating new person with envConfig:',
+      envConfig
+    );
+
+    const person = new TinnyPerson({
+      privateKey,
+      envConfig,
+    });
+
+    await person.spawn();
+
+    this.world.set(name, person);
+
+    return person;
+  }
+
+  /**
+   * Retrieves a person from the world by their name.
+   * @param name - The name of the person to retrieve.
+   * @returns The person object if found, or undefined if not found.
+   */
+  getPerson(name: string) {
+    return this.world.get(name);
+  }
+
+  /**
+   * Creates a random person.
+   * @returns A promise that resolves to the created person.
+   */
+  async createRandomPerson() {
+    return await this.createNewPerson('Alice');
```

Comment:
All random person's are Alice? Can we put a random number on the name or something just for debugging's sake? :) Also, if I call this method multiple times, it's going to overwrite the _same_ user on the `world` object every time I call it, right? That seems like not what we want?